docs/upgrade/UPGRADE-v7.3.2-dev.md

```diff
@@ -59,4 +59,87 @@ There you can find links to upgrade notes for other versions too.
     - restart you Nginx for the changes to take effect
         - for example, run `docker-compose up -d --force-recreate webserver` to restart the container in Docker Compose
 
-[shopsys/framework]: https://github.com/shopsys/framework
\ No newline at end of file
+### Frontend changes
+- update your `project-base/src/Shopsys/ShopBundle/Resources/views/Front/Content/Brand/list.html.twig` ([#1371](https://github.com/shopsys/shopsys/pull/1371))
```

Comment:
This should be done as one instruction with link to the PR and more sub-instructions
something like 
```
 - fix brand label when the brand name is long or have two lines  ([#1371](https://github.com/shopsys/shopsys/pull/1371))
    - update your `project-base/src/Shopsys/ShopBundle/Resources/views/Front/Content/Brand/list.html.twig` 
    ...
    - update your `project-base/src/Shopsys/ShopBundle/Resources/styles/front/common/components/list/images.less` like this: ...
```

Reply:
Rewrited according your notes. Thx for review